cmd/renterd/main.go

```diff
@@ -104,6 +104,11 @@ var (
 			ScannerNumThreads:              100,
 			MigratorParallelSlabsPerWorker: 1,
 		},
+		S3: config.S3{
+			Enabled:     false,
+			DisableAuth: false,
```

Comment:
Does it make sense to turn this into `AuthEnabled` to avoid the double negation with our `!disabled` checks? I don't really mind though.